<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HypothesisTestDock</class>
 <widget class="QWidget" name="HypothesisTestDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>1115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="23" column="3" colspan="3">
    <widget class="QLineEdit" name="leMuo"/>
   </item>
   <item row="1" column="3" colspan="3">
    <widget class="ResizableTextEdit" name="teComment"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QLabel" name="lCalculateStats">
     <property name="text">
      <string>Calculate Statistics 
From Spreadsheet</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0" colspan="2">
    <widget class="QLabel" name="lPopulationSigma">
     <property name="text">
      <string>Sigma</string>
     </property>
    </widget>
   </item>
   <item row="24" column="3" colspan="3">
    <widget class="QLineEdit" name="leAlpha"/>
   </item>
   <item row="21" column="3" colspan="3">
    <widget class="QFrame" name="frameNullHypothesis">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rbH0OneTail1">
        <property name="text">
         <string>mu &lt;= muo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbH0OneTail2">
        <property name="text">
         <string>mu &gt;= muo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbH0TwoTail">
        <property name="text">
         <string>mu == muo</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="16" column="0" colspan="2">
    <widget class="QLabel" name="lEqualVariance">
     <property name="text">
      <string>Equal Variance</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QLabel" name="lCol1">
     <property name="text">
      <string>Independent Var. 1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lTest_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Test</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0" colspan="2">
    <widget class="QLabel" name="lCategorical">
     <property name="text">
      <string>Independent Var. 1 Cateogical</string>
     </property>
    </widget>
   </item>
   <item row="16" column="5">
    <widget class="QPushButton" name="pbLeveneTest">
     <property name="text">
      <string>Levene's Test</string>
     </property>
    </widget>
   </item>
   <item row="26" column="5" rowspan="2">
    <widget class="QPushButton" name="pbPerformTest">
     <property name="text">
      <string>Recalculate</string>
     </property>
    </widget>
   </item>
   <item row="18" column="5">
    <widget class="QLineEdit" name="lePopulationSigma"/>
   </item>
   <item row="0" column="3" colspan="3">
    <widget class="QLineEdit" name="leName"/>
   </item>
   <item row="23" column="0" colspan="2">
    <widget class="QLabel" name="lMuo">
     <property name="text">
      <string>muo</string>
     </property>
    </widget>
   </item>
   <item row="15" column="3" colspan="3">
    <widget class="QComboBox" name="cbCol2"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lSpreadsheet">
     <property name="text">
      <string>Spreadsheet:</string>
     </property>
    </widget>
   </item>
   <item row="24" column="0" colspan="2">
    <widget class="QLabel" name="lAlpha">
     <property name="text">
      <string>alpha</string>
     </property>
    </widget>
   </item>
   <item row="10" column="3" colspan="3">
    <widget class="QCheckBox" name="chbCalculateStats">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lComment">
     <property name="text">
      <string>Comment:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lTestType">
     <property name="text">
      <string>Sub-type</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="QLabel" name="lVariables">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Variables</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="2">
    <widget class="QLabel" name="lCol3">
     <property name="text">
      <string>Independent Var. 2</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>32</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="17" column="3">
    <widget class="QCheckBox" name="chbCategorical">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="22" column="3" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rbH1OneTail1">
        <property name="text">
         <string>mu &gt; muo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbH1OneTail2">
        <property name="text">
         <string>mu &lt; muo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbH1TwoTail">
        <property name="text">
         <string>mu != muo</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="21" column="0" colspan="2">
    <widget class="QLabel" name="lH0">
     <property name="text">
      <string>Null</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="13" column="3" colspan="3">
    <widget class="QComboBox" name="cbCol1"/>
   </item>
   <item row="22" column="0" colspan="2">
    <widget class="QLabel" name="lH1">
     <property name="text">
      <string>Alternate</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lData">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Data</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0" colspan="2">
    <widget class="QLabel" name="lCol2">
     <property name="text">
      <string>Variable</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>13</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="19" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>32</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="18" column="3">
    <widget class="QCheckBox" name="chbPopulationSigma">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="27" column="3" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>150</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="16" column="3">
    <widget class="QCheckBox" name="chbEqualVariance">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="25" column="0" rowspan="2" colspan="6">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="3" colspan="3">
    <widget class="QComboBox" name="cbTestType"/>
   </item>
   <item row="28" column="1">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>18</width>
       <height>38</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="3" colspan="3">
    <widget class="QComboBox" name="cbTest"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lTest">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>32</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>32</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="14" column="3" colspan="3">
    <widget class="QComboBox" name="cbCol3"/>
   </item>
   <item row="20" column="0" colspan="2">
    <widget class="QLabel" name="lHypothesis">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Hypothesis</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="3">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ResizableTextEdit</class>
   <extends>QTextEdit</extends>
   <header>src/kdefrontend/widgets/ResizableTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
